Changes.md

```diff
@@ -1,5 +1,12 @@
 <!-- next-header -->
 
+## 0.5.1
+
+- Added a new labels feature. This allows you to group commands in your config
+  file by assigning one or more `labels` in their config. Then when running
+  `precius`, you can run commands with a specific label: `precious lint
```

Comment:
typo: `precius` -> `precious`

Reply:
Thanks! I fixed this and all the other things you noted.